### ISSUE TITLE
Questions about the logit function

### ISSUE THREAD

**Issue by sydney0zq** [seoungwugoh/RGMP#8]
In your RGMP paper Sec 3.3, you write the soft logit function. But in your code [logit](https://github.com/seoungwugoh/RGMP/blob/7461148cc2be01c367571fa6141ef769c9a5bb70/run.py#L102), I am a little confused about it.

Questions:
1. In your training, you view the background as one instance and then propagate it? If so, the probability (Mask) of the background is the sum of other instance labels, which really confuses me.
2. Could you please provide more details in training? (How to logit function with softmax is being used, is it just like the inference stage?) 

Thanks in advance!

Sorry for my mistake.
